source/plugin/data/custom/datamanipulators.rst

```diff
@@ -144,6 +150,49 @@ change is made to the format of your serialized data, and use :ref:`content-upda
         return container;
     }
 
+Registration
+============
+
+Registering your ``DataManipulator`` allows it to be accessible by Sponge and by other plugins in a generic way. The
+game/plugin can create copies of your data and serialize/deserialize your data without referencing any of your classes
+directly.
+
+To register a ``DataManipulator`` Sponge has the :javadoc:`DataRegistration#builder()` helper. This will build a
+:javadoc:`DataRegistration` and automatically register it.
+
+
+.. note::
+
+    Due to the nature of Data, you *must* register your ``DataManipulator`` during initialization - generally by
+    listening to :javadoc:`GameInitializationEvent` such as the below. If you try to register a ``DataManipulator`` once
```

Comment:
"such as the below" needs fixing ... please either add the word "example" or remove the "the".